sparse/tpls/KokkosSparse_sptrsv_symbolic_tpl_spec_avail.hpp

```diff
@@ -20,11 +20,47 @@
 namespace KokkosSparse {
 namespace Impl {
 // Specialization struct which defines whether a specialization exists
-template <class KernelHandle, class RowMapType, class EntriesType>
+template <class KernelHandle, class RowMapType, class EntriesType,
+          class ValuesType>
 struct sptrsv_symbolic_tpl_spec_avail {
   enum : bool { value = false };
 };
 
+// cuSPARSE
+#ifdef KOKKOSKERNELS_ENABLE_TPL_CUSPARSE
+
+#define KOKKOSSPARSE_SPTRSV_SYMBOLIC_TPL_SPEC_AVAIL_CUSPARSE(SCALAR, LAYOUT,   \
+                                                             MEMSPACE)         \
+  template <>                                                                  \
+  struct sptrsv_symbolic_tpl_spec_avail<                                       \
+      KokkosKernels::Experimental::KokkosKernelsHandle<                        \
+          const int, const int, const SCALAR, Kokkos::Cuda, MEMSPACE,          \
+          MEMSPACE>,                                                           \
+      Kokkos::View<                                                            \
+          const int *, LAYOUT, Kokkos::Device<Kokkos::Cuda, MEMSPACE>,         \
+          Kokkos::MemoryTraits<Kokkos::Unmanaged | Kokkos::RandomAccess> >,    \
+      Kokkos::View<                                                            \
+          const int *, LAYOUT, Kokkos::Device<Kokkos::Cuda, MEMSPACE>,         \
+          Kokkos::MemoryTraits<Kokkos::Unmanaged | Kokkos::RandomAccess> >,    \
+      Kokkos::View<                                                            \
+          const SCALAR *, LAYOUT, Kokkos::Device<Kokkos::Cuda, MEMSPACE>,      \
+          Kokkos::MemoryTraits<Kokkos::Unmanaged | Kokkos::RandomAccess> > > { \
+    enum : bool { value = true };                                              \
+  };
+
+KOKKOSSPARSE_SPTRSV_SYMBOLIC_TPL_SPEC_AVAIL_CUSPARSE(float, Kokkos::LayoutLeft,
+                                                     Kokkos::CudaSpace)
+KOKKOSSPARSE_SPTRSV_SYMBOLIC_TPL_SPEC_AVAIL_CUSPARSE(double, Kokkos::LayoutLeft,
+                                                     Kokkos::CudaSpace)
+KOKKOSSPARSE_SPTRSV_SYMBOLIC_TPL_SPEC_AVAIL_CUSPARSE(Kokkos::complex<float>,
+                                                     Kokkos::LayoutLeft,
+                                                     Kokkos::CudaSpace)
+KOKKOSSPARSE_SPTRSV_SYMBOLIC_TPL_SPEC_AVAIL_CUSPARSE(Kokkos::complex<double>,
+                                                     Kokkos::LayoutLeft,
+                                                     Kokkos::CudaSpace)
```

Comment:
```CudaUVMSpace``` is supported in ```sparse/tpls/KokkosSparse_sptrsv_solve_tpl_spec_decl.hpp```. I think it should be added here too.

Reply:
Ah fair enough, thanks for catching this, I forgot about it x)